modules/ROOT/pages/security/single-sign-on.adoc

```diff
@@ -0,0 +1,85 @@
+[[aura-reference-security]]
+= Single Sign-On (SSO)
+:description: SSO allows you to log in to the Aura Console using their company IdP credentials.
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+label:AuraDB-Business-Critical[]
+label:AuraDS-Enterprise[]
+
+== SSO levels
+
+Organization admins can configure SSO on organization level and project level.
+SSO is a log-in method. 
+Access, roles, and permissions are dictated by role-based access control (RBAC).
+
+* *Organization-level:* Allows org admins to control how users log in when they are trying to access the organization. It is a log-in method for the Aura console.
+
+* *Project-level:*  Impacts new database instances created within that project.
+It ensures users logging in with SSO have access to the database instances within the project.
+It depends on RBAC if the user can access and view or modify data within the database instances themselves.
```

Comment:
```suggestion
It depends on RBAC if the user can access and view or modify data within the instances themselves.
```
To say _database instance_ can be ambiguous since it is clustering terminology and that's not what we are referring to here.